style.css

```diff
@@ -0,0 +1,93 @@
+body{
```

Comment:
Before curly bracket add space. Also, go through this file and add spaces before some curly brackets and after some colons too.